### PR TITLE
Create a single source of series.

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
QA:
- do a search
- the filter on the top right should display a full list of series
- drop a local charm on the canvas
- the provided list of series should be the same as the search filter

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3622/
Test PASSed.

File: jujugui/static/gui/src/app/views/utils.js [hunk 1]
```diff
@@ -36,6 +36,38 @@ YUI.add('juju-view-utils', function(Y) {
...
+
+     @method getSeriesList
+     @return {Object} A collection of series.
+   */
+  utils.getSeriesList = function() {
+    // For a list of supported series in Juju see:
+    // https://github.com/juju/charmstore/blob/v5-unstable/internal/
+    // series/series.go#L37
+    return {
+      precise: {name: 'Precise Pangolin 12.04 LTS'},
```
Comment (fabricematrat) [inline jujugui/static/gui/src/app/views/utils.js]: I think the list should be the same than here 
https://github.com/juju/charmstore/blob/v5-unstable/internal/series/series.go#L42
which also means one day we need to get that list from an endpoint in charmstore instead.

**Comment by fabricematrat** [comment]
:+1:  with one change requested for the list. code :ok:

**Comment by hatched** [comment]
:+1: QA :ok:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-gui/3629/
Test PASSed.

**Comment by huwshimi** [comment]
Thanks! :shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**